exercises/practice/game-of-life/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.5-bin.zip
```

Comment:
Other practice exercises are using Gradle wrapper version 8.7, so I was wondering how you created the wrapper. If you copied this from somewhere in this repo it means we missed something in the latest upgrades, which should be addressed.

If you created the wrapper yourself, please make sure to use the same version as the other exercises. This document explains how to update/sync Gradle versions: https://github.com/exercism/java/blob/main/reference/how-to-update-gradle.md

Reply:
I followed the steps and the Contributing.md file, and copied `exercise-template`. This is coming from that. Will update the Gradle version.

Reply:
Ah, so we missed the exercise template when updating the Gradle version. Good to know!

Perhaps you can also update the exercise template itself in this PR and add a note to the document on how to update Gradle that the template should be updated as well?

Reply:
Sure, will update